stream_alert/rule_processor/rules_engine.py

```diff
@@ -443,10 +444,12 @@ def rule_analysis(record, rule, payload, alerts):
             if StreamRules.check_alerts_duplication(record, rule, alerts):
                 return
 
-            LOGGER.info('Rule [%s] triggered an alert on log type [%s] from entity \'%s\' '
-                        'in service \'%s\'', rule.rule_name, payload.log_source,
+            alert_id = str(uuid.uuid4())  # Random unique alert ID
+            LOGGER.info('Rule [%s] triggered alert [%s] on log type [%s] from entity \'%s\' '
```

Comment:
Isn't this a redundant logger message since `main.py` has a similar one when dispatching?

Reply:
This log happens when the alert is built in the rule processor, the `main.py` you commented on above is in the alert processor. I like having both because then the alert ID is in the logs of both processors (and the rule processor / alert processor will be completely decoupled soon)